openedx/features/personalized_learner_schedules/call_to_action.py

```diff
@@ -94,4 +101,14 @@ def _make_reset_deadlines_cta(xblock):
             'description': _('To participate in this assignment, the suggested schedule for your course needs '
                              'updating. Don’t worry, we’ll shift all the due dates for you and you won’t lose '
                              'any of your progress.'),
+
         }
+
+        if is_learning_mfe:
+            cta_data['event_data'] = {
+                'body': _('View all dates'),
```

Comment:
Don't we need a link here? Otherwise the MFE is going to have to know what kind of message it's getting in order to insert the right link.

And aren't we going to have to give it the POST url? With a dictionary of form_values? Otherwise the MFE is going to have to hardcode where & how to POST.